src/server.js

```diff
@@ -20,7 +19,7 @@ const server = new Server(
   },
 )
 
-const WEBSITE_GENERATION_TOOL: Tool = {
```

Comment:
I still havent tested this update as mcp. I am forgetting how to test a development version. I will work with @zhadaev on Monday to test this before we this update. We will also have to update our npm package.